mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -465,6 +465,36 @@ private[python] class PythonMLLibAPI extends Serializable {
   }
 
   /**
+   * Java stub for Python mllib LDA.run()
+   */
+  def trainLDAModel(
+      data: JavaRDD[LabeledPoint],
```

Comment:
Note from discussion: Let's avoid use of LabeledPoint.  We can use fromTuple2RDD to handle the pair of values.